code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/deployment/impl/DeploymentManagerImpl.java

```diff
@@ -268,12 +269,26 @@ public void doWithLockNoResult() {
 
     @Override
     public void activateGlobalServicesForHost(long accountId, long hostId) {
+
```

Comment:
Yup.  Just tested it out with breakpoints set in:
- AgentResourcesMonitorImpl.java where we create the host DB entry
- and the activate(service) call in the DeferredServiceActivate class

activate(service) doesn't get executed until after we've created the host DB entry